### ISSUE TITLE
Ditto /PlainTextPaste command-line option doesn't output the correct clipboard content

### ISSUE THREAD

**Issue by samegek** [sabrogden/Ditto#466]
Description:

When using the 
`Ditto.exe /PlainTextPaste`
 command-line option to output the current clipboard content, the output is incorrect after deleting the latest copied item from the Ditto clipboard history.

For example:

1. Copy "hello, world".
2. Execute 
`Ditto.exe /PlainTextPaste`
, the output is "hello, world".
3. Copy "foo, bar".
4. Execute 
`Ditto.exe /PlainTextPaste`
, the output is "foo, bar".
5. Delete "foo, bar" from Ditto clipboard history.
6. Execute 
`Ditto.exe /PlainTextPaste`
, the output is still "foo, bar", but it should be "hello, world".
7. However, using the default paste shortcut works correctly and outputs "hello, world".
Expected behavior:
When executing 
`Ditto.exe /PlainTextPaste`
, the output should always match the latest clipboard content, even after deleting an item from Ditto clipboard history.

Steps to reproduce:

1. Copy some text.
2. Execute 
`Ditto.exe /PlainTextPaste`
, verify that the output matches the clipboard content.
3. Copy some more text.
4. Execute 
`Ditto.exe /PlainTextPaste`
, verify that the output matches the latest clipboard content.
5. Delete the latest copied item from Ditto clipboard history.
6. Execute 
`Ditto.exe /PlainTextPaste`
, verify that the output still matches the latest clipboard content.
Environment:

Ditto Version 03.24.214.OO 32bit
Sqlite Version 3.33.0
win10

**Comment by sabrogden** [comment]
This works the way it's intended to work.  /PlainTextPaste goes off of the current clipboard contents, not the top or any clip in Ditto.

**Comment by samegek** [comment]
but default i can easily get current clipboard by other way, but i cant get top clip in ditto by command line.
i need a interface get ditto top clip

**Comment by Chealer** [comment]
@samegek: I agree that the description ("Pastes current clipboard data as plain text ") would make me think it would paste Ditto's current clip. "Pastes Windows clipboard contents" would be much clearer.

But in any case, `/PlainTextPaste` is definitely not designed for what you want (output), since it pastes. You may be able to achieve what you want with the CopyQ clipboard manager using `.\copyq.exe select x`  followed by `.\copyq.exe clipboard | Write-Output`.

<details>
  <summary>License</summary>
This comment and any other from myself on this ticket is offered under the terms of CC0 1.0.
</details>
